core/src/main/kotlin/gui/Menu.kt

```diff
@@ -0,0 +1,128 @@
+package gui
+
+import com.badlogic.gdx.Game
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.Screen
+import com.badlogic.gdx.graphics.GL20
+import com.badlogic.gdx.graphics.Texture
+import com.badlogic.gdx.graphics.g2d.Sprite
+import com.badlogic.gdx.graphics.g2d.SpriteBatch
+import com.badlogic.gdx.graphics.g2d.TextureRegion
+import com.badlogic.gdx.scenes.scene2d.InputEvent
+import com.badlogic.gdx.scenes.scene2d.InputListener
+import com.badlogic.gdx.scenes.scene2d.Stage
+import com.badlogic.gdx.scenes.scene2d.ui.ImageTextButton
+import com.badlogic.gdx.scenes.scene2d.ui.Skin
+import com.badlogic.gdx.scenes.scene2d.utils.TextureRegionDrawable
+import com.badlogic.gdx.utils.viewport.ScreenViewport
+import kotlin.system.exitProcess
+
+
+class Menu(val game: Game) : Screen {
+
+    lateinit var stage: Stage
+    lateinit var batch: SpriteBatch
+    lateinit var checker: Texture
+    lateinit var title: Texture
+    lateinit var checkerSprite: Sprite
+    lateinit var titleSprite: Sprite
+
+    private fun setNewGame() {
+        val skinNew = Skin(Gdx.files.internal(GUIConstants.textSkinPath))
+        val newGame = ImageTextButton(GUIConstants.newGameText, skinNew, GUIConstants.styleName)
```

Comment:
Game should be fully initialized inside its constructor

Reply:
done